src/common/components/signUpModal/signUpModal.component.js

```diff
@@ -476,28 +505,71 @@ class SignUpModalController {
   }
 
   injectErrorMessages (errors = this.signUpErrors) {
+    this.clearInjectedErrorMessages()
+
+    if (!errors) {
+      return
+    }
+
     // Inject error messages into the form since errors are cleared when switching steps/rerendering.
-    errors?.forEach(error => {
+    errors.forEach(error => {
       const field = document.querySelector(`${inputFieldErrorSelectorPrefix}${error.property.replace(/\./g, '\\.')}`)
       if (field) {
-        // Only add an error message if it doesn't already exist
-        const existingErrorParentElement = field.parentNode.querySelector('.okta-form-input-error')
-        const errorText = `${errorIconHtml} ${error.errorSummary}`
-        if (!existingErrorParentElement || existingErrorParentElement.innerHTML !== errorText) {
-          const errorElement = document.createElement('div')
-          errorElement.classList.add('okta-form-input-error', 'o-form-input-error', 'o-form-explain')
-          field.parentNode.classList.add('o-form-has-errors')
-          errorElement.setAttribute('role', 'alert')
-          errorElement.innerHTML = errorText
-          field.parentNode.appendChild(errorElement)
-        }
+        const errorElement = document.createElement('div')
```

Comment:
Why did you remove the code that ensures we only add the error to the field once?

Reply:
I added `this.clearInjectedErrorMessages()` to the top of the function to delete all error messages.

Reply:
Awesome!